assayist/common/models/content.py

```diff
@@ -39,8 +38,9 @@ class Artifact(AssayistStructuredNode):
 
     architecture = StringProperty()
     archive_id = StringProperty(required=True, unique_index=True)
-    checksum = StringProperty()
     filename = StringProperty()
+    # A one-word description of the type of file this describes (to aid in filtering)
+    type = StringProperty(required=True)
```

Comment:
This is probably just a nitpick, but could we consider renaming all the `type` attributes to something that doesn't shadow a Python built-in? I think using a trailing underscore is the usual convention: `type_`.

Reply:
This is a class property so I think its scope is always clear when it is used.